code/io/tree_binary.c

```diff
@@ -20,13 +20,15 @@ FILE *load_fd;
 
 // External Functions //
 
+#define MAX_BUF_SIZE (3*MAX_STRING_LEN+40)
```

Comment:
Good idea. Should we check whether `MAX_BUF_SIZE` is defined? Something along the lines of 

#ifndef MAX_BUF_SIZE
#define MAX_BUF_SIZE ...
#endif

Reply:
OK, but in that case there should be a consistent "universal" value is - the safe value would obviously be the biggest. My version of the pull request currently gives
````shell
$ grep --color -rn "define.*MAX_BUF_SIZE " code/*
code/core_io_tree.c:19:#define MAX_BUF_SIZE (3*MAX_STRING_LEN+20)
code/core_save.c:17:#define MAX_BUF_SIZE (2*MAX_STRING_LEN+40)
code/io/tree_binary.c:23:#define MAX_BUF_SIZE (3*MAX_STRING_LEN+40)
````
which is not globally consistent.

So 
````C
#ifndef MAX_BUF_SIZE
#define MAX_BUF_SIZE (3*MAX_STRING_LEN+40)
#endif
````
would seem the safest in all three places.